core/src/main/java/org/nanohttpd/protocols/http/response/MimeTypes.java

```diff
@@ -0,0 +1,75 @@
+package org.nanohttpd.protocols.http.response;
+
+/*
+ * #%L
+ * NanoHttpd-Core
+ * %%
+ * Copyright (C) 2012 - 2016 nanohttpd
+ * %%
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the nanohttpd nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software without
+ *    specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+ * OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+public enum MimeTypes {
+
+  TEXT("text/plain"),
+  HTML("text/html"),
+  CSS("text/css"),
+  JPEG("image/jpeg"),
+  PNG("image/png"),
+  GIF("image/gif"),
+  BMP("image/bmp"),
+  WEBP("image/webp"),
+  MPEG("audio/mpeg"),
+  OGG("audio/ogg"),
+  MIDI("audio/midi"),
+  WEBM("audio/webm"),
+  WAV("audio/wav"),
+  MP4("video/mp4"),
+  JSON("application/json"),
+  JAVASCRIPT("application/javascript"),
+  ECMASCRIPT("application/ecmascript"),
+  OCTETSTREAM("application/octet-stream"),
+  XML("application/xml"),
+  XHTMLXML("application/xhtml+xml"),
+  VNDMSPowerPoint("application/vnd.mspowerpoint"),
+  PDF("application/pdf");
+
+  private String value;
+
+  MimeTypes(String value) {
+    this.value = value;
+  }
+
+  public String getValue() {
+    return value;
```

Comment:
is there a reason to have `getValue()` when there is already `toString()`?

Reply:
Yes. When you have enums with
```java
public enum EFooBar{
    X("foo", 10),
    Y("bar", 5),
    Z("baz", 0);
}
```
`toString()` will return "X" / "Y" / "Z". If you want to distinguish between the format and the `Content-Type` you need to use a separate method.
Arguably we should rename `getValue()` to `getContentType()` though, because that's what it is.

Reply:
Umm, I will change getValue but i will keep toString the same